.github/scripts/translate-strings.py

```
@@ -0,0 +1,229 @@
+            time.sleep(self.rate_limit_delay)  # Rate limiting
+        
+        return results
+
+class StringTranslator:
+    def __init__(self, api_url: str, api_key: str = None):
+        self.client = LibreTranslateClient(api_url, api_key)
+        
+    def parse_strings_xml(self, xml_file: str) -> Dict[str, str]:
+        """Parse strings XML file and return key-value pairs."""
+        try:
+            tree = ET.parse(xml_file)
```

Ruff (0.14.14)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)